### DISCUSSION TITLE
Add "beta" to list of known subdomains for testing

### DISCUSSION THREAD

**Discussion by nathansnelgrove** [craftcms/cms#10081] [Ideas] [answered]
Hey there,

I've had a couple clients in the past year who prefer having their test sites set up on something beta.theirdomain.com, instead of staging.theirdomain.com. The problem with the beta thing is that we can't activate plugin trials on their test sites. 

In theory, we could just use staging as the subdomain, but staging is often taken for a dedicated app or their already-existing website.

It'd be great if the licensing system recognized subdomains with the word "beta" in them as test sites as well.

**Comment by brandonkelly** [comment, marked as answer]
`beta` is commonly used for public “betas” of new site versions, similar to `new.example.com`, so we aren’t going to allow that.

**Comment by nathansnelgrove** [comment]
Hadn’t considered that — makes sense! Cheers.

**Reply by vitalijalbu** [reply]
is there a way?? i have a 3rd domain for a beta website to test, but cannot due to license key.

**Reply by brandonkelly** [reply]
Use any of the supported dev subdomains instead. They could be mixed with `beta`, e.g. `staging.beta.example.tld`.

**Reply by vitalijalbu** [reply]
**demo** subdomain cannot be valid?

**Reply by brandonkelly** [reply]
Sorry, no. High chance that would be used for a production site.
